src/protocol/mod.rs

```diff
@@ -702,6 +715,24 @@ mod load {
             Ok(())
         }
     }
+
+    #[test]
+    fn allows_to_specify_the_local_folder_tree() -> R<()> {
+        assert_eq!(
+            test_parse_one(
+                r##"
+                    |protocol:
+                    |  - /bin/true
+                    |filetree:
+                    |  - /usr/local/bin/unrar
+                "##
+            )?
+            .mocked_files
+            .map(|mocked_file| String::from_utf8(mocked_file).unwrap()),
```

Comment:
Maybe `.map(String::from_utf8_lossy)`?

Reply:
tried that, was not able to get the types to work out by just passing the function, but 
```rust
.map(|path| String::from_utf8_lossy(&path).to_string()),
```
works. 

if i understand correctly, `&path` takes `Vec<u8>` => `&[u8]` to satisfy `from_utf8_lossy`'s type signature. is there a way around having to explicitly do this? also, `from_utf8_lossy` gives me back a 🐮 `Cow<_, str>` which i think i have to convert in order to compare.

Reply:
Yeah, I think this is all necessary. Instead of `to_string` you could use `into_owned`, I think, which would work for all `Cow`s, not just for `Cow`s of strings.